include/OpenColorIO/OpenColorTransforms.h

```diff
@@ -563,8 +563,63 @@ OCIO_NAMESPACE_ENTER
     
     //!cpp:function::
     extern OCIOEXPORT std::ostream& operator<< (std::ostream&, const FileTransform&);
+
+
+    //!rst:: //////////////////////////////////////////////////////////////////
+
+    //!cpp:class:: Provides a set of hard-coded algorithmic building blocks 
+    // that are needed to accurately implement various common color transformations.
+    //
+    //
+    class OCIOEXPORT FixedFunctionTransform : public Transform
+    {
+    public:
+        //!cpp:function::
+        static FixedFunctionTransformRcPtr Create();
+        
+        //!cpp:function::
+        virtual TransformRcPtr createEditableCopy() const;
+        
+        //!cpp:function::
+        virtual TransformDirection getDirection() const;
+        //!cpp:function::
+        virtual void setDirection(TransformDirection dir);
+
+        //!cpp:function:: Will throw if data is not valid.
+        virtual void validate() const;
+
+        //!cpp:function::
+        virtual FixedFunctionStyle getStyle() const;
+        //!cpp:function:: Select which algorithm to use.
+        virtual void setStyle(FixedFunctionStyle style);
+
+        //!cpp:function::
+        size_t getNumParams() const;
+        //!cpp:function:: Set the parameters (for functions that require them).
```

Comment:
Are the required params for each FixedFunction style documented somewhere (outside of looking at the lower level implementation classes)? AllocationTransform, which shares a similar interface, was never super clear in regards to how many and what the params are for each allocation option. With this Transform, there are many more styles, so I would propose a brief overview of the params for each here in OpenColorTransforms.h, or along with the enum in OpenColorTypes.h, with the intent being to be visible to the API docs.

I also think the explanation about which ACES versions the specific functions are defined in (like in FixedFunctionOpData.h) would be useful in the API docs as well, to be obvious about "03" referring to ACES 0.3/0.7 and "10" to ACES 1.0.

Reply:
Good catch. We will improve the OCIO public API documentation.

Reply:
Done.